crypto/rand/rand_unix.c

```diff
@@ -29,19 +29,19 @@
 #if defined(__OpenBSD__) || defined(__NetBSD__)
 # include <sys/param.h>
 #endif
-
-#if defined(OPENSSL_SYS_UNIX) || defined(__DJGPP__)
-# include <sys/types.h>
-# include <sys/stat.h>
-# include <fcntl.h>
-# include <unistd.h>
-# include <sys/time.h>
+#if !defined(OPENSSL_SYS_VXWORKS)
```

Comment:
could you change this to
`#if (defined(OPENSSL_SYS_UNIX) && !defined(OPENSSL_SYS_VXWORKS)) || defined(__DJGPP__)`
instead please?

Reply:
I mean there must be a way to get a manageable diff here.  this changes pretty much everything in here.